src/components/Sidebar/NavItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, Text, Icon, Link, Menu, MenuButton } from "@chakra-ui/react";
+
+const NavItem = ({ navSize, title, icon }: any) => {
+  return (
+    <Flex
+      ml={navSize === "large" ? "-220%" : "-380%"}
+      mt={navSize === "large" ? "20%" : "20%"}
+      flexDir="column"
+      w="200%"
+      alignItems={navSize === "small" ? "center" : "flex-start"}
+    >
+      <Menu placement="right">
+        <Link
+          href={
+            title === "List"
```

Comment:
extract to getLink function, it's hard to read, without nester ternar expression